src/core/gpu_vk.c

```diff
@@ -144,7 +147,8 @@ static struct {
 enum { CPU, GPU };
 enum { LINEAR, SRGB };
 
-static void hash32(uint32_t* hash, void* data, uint32_t size);
+// Had to rename vHash to include util.h-- can be removed before merge
+static void vHash32(uint32_t* hash, void* data, uint32_t size);
```

Comment:
util.h also has a hash32() as a #define. Although the lovrLogs in this patch are not necessary and the util.h include can be removed, probably it's worth it to keep this function name changed to avoid clashes if util.h is ever added for a good reason.